### ISSUE TITLE
Enabling "Compressed JSON" option entirely deletes content of a drawing 

### ISSUE THREAD

**Issue by DraggonFantasy** [zsviczian/obsidian-excalidraw-plugin#813]
**Describe the bug**
I had "Compressed JSON" option disabled in settings all the time I was using Excalidraw plugin. And I had drawn plenty of images during that time. Then I decided to try to enable "Compressed JSON" and see the difference in size of files/search etc. I enabled it and after that some of my drawing were broken - they were cleared right after I opened them

**To Reproduce**
1. Create a new Obsidian vault
2. Install Excalidraw there
3. Download the attached drawing and put it inside the vault
4. Open it
5. Go to Excalidraw settings
6. Enable "Compressed JSON"
7. Close the pane with drawing
8. Open the drawing again

**Expected behavior**
The drawing is opened the same as it was

**Actual behaviour**
Blank page is opened and the contents of the drawing is irretrievably lost

**Screenshots**
Drawing before enabling "Compressed JSON":
![image](https://user-images.githubusercontent.com/19591977/190992280-23fd0319-91a3-4ce8-b740-7002a308f79b.png)
![image](https://user-images.githubusercontent.com/19591977/190992487-1d06f766-cc4e-4b85-9a32-d79721578928.png)

The same drawing after enabling "Compressed JSON":
![image](https://user-images.githubusercontent.com/19591977/190992635-34135aab-4ba2-47e6-91b8-95898e0b7e64.png)
![image](https://user-images.githubusercontent.com/19591977/190992972-70ff4805-1b2d-478f-92c1-1a5f08afa350.png)

[Японские слова.excalidraw.md](https://github.com/zsviczian/obsidian-excalidraw-plugin/files/9597899/excalidraw.md)

**Environment (please complete the following information):**
 - OS including version: Kubuntu 20.04
 - Plugin version: 1.7.20
 - Obsidian version: 0.15.9

**The reproduceable Excalidraw drawing is attached to this issue**

**Comment by zsviczian** [comment]
I am sorry for the loss of file. I hope you have some backup set up to avoid information loss.

Are you able to reproduce this issue, or was this a once off event? I wasn't able to reproduce the problem (yet).

**Comment by DraggonFantasy** [comment]
Yes, I've made a backup before changing the settings, so I recovered it after the loss :slightly_smiling_face: 
Yes, for me its reproducible every time with this file I've attached to the issue. For some reason not every file is being cleared after  switching the setting, but this file is 100% reproducible for me. Not sure about the difference between files which are cleared and which are not yet, but maybe I'll try to figure this out.

**Comment by DraggonFantasy** [comment]
I've reinstalled Obsidian with latest installer from their site and the problem seems to be resolved. At least it doesn't reproduce anymore with this file
